data/brands/shop/butcher.json

```diff
@@ -382,6 +382,20 @@
         "shop": "butcher"
       }
     },
+    {
+      "displayName": "ბიუბიუ",
+      "locationSet": {"include": ["ge"]},
+      "tags": {
+        "brand": "ბიუბიუ",
```

Comment:
Please add a `brand:wikidata` tag, creating a new Wikidata item if necessary. For information on editing Wikidata, please see https://github.com/osmlab/name-suggestion-index/wiki/Editing-Wikidata

Reply:
Added